client/components/search/index.js

```diff
@@ -71,55 +78,111 @@ class Search extends Component {
 		}
 	}
 
+	renderTags() {
+		const { selected } = this.props;
+		return selected.length ? (
+			<div className="woocommerce-search__token-list">
+				{ selected.map( ( item, i ) => {
+					const screenReaderLabel = sprintf(
+						__( '%1$s (%2$s of %3$s)', 'wc-admin' ),
+						item.label,
+						i,
+						selected.length
+					);
+					return (
+						<Tag
+							key={ item.id }
+							id={ item.id }
+							label={ item.label }
+							remove={ this.removeResult }
+							removeLabel={ __( 'Remove product', 'wc-admin' ) }
+							screenReaderLabel={ screenReaderLabel }
+						/>
+					);
+				} ) }
+			</div>
+		) : null;
+	}
+
+	onFocus() {
+		this.setState( { isActive: true } );
+	}
+
+	onBlur() {
+		this.setState( { isActive: false } );
+	}
+
 	render() {
 		const autocompleter = this.getAutocompleter();
-		const { placeholder, selected } = this.props;
-		const { value = '' } = this.state;
+		const { placeholder, inlineTags, selected, instanceId } = this.props;
+		const { value = '', isActive } = this.state;
 		const aria = {
 			'aria-labelledby': this.props[ 'aria-labelledby' ],
 			'aria-label': this.props[ 'aria-label' ],
 		};
 		return (
 			<div className="woocommerce-search">
+				<label htmlFor={ `search-input-${ instanceId }` } className="screen-reader-text">
+					{ placeholder || __( 'Add item', 'wc-admin' ) }
+				</label>
 				<Gridicon className="woocommerce-search__icon" icon="search" size={ 18 } />
 				<Autocomplete completer={ autocompleter } onSelect={ this.selectResult }>
-					{ ( { listBoxId, activeId, onChange } ) => (
-						<input
-							type="search"
-							value={ value }
-							placeholder={ placeholder }
-							className="woocommerce-search__input"
-							onChange={ this.updateSearch( onChange ) }
-							aria-owns={ listBoxId }
-							aria-activedescendant={ activeId }
-							{ ...aria }
-						/>
-					) }
-				</Autocomplete>
-				{ selected.length ? (
-					<div className="woocommerce-search__token-list">
-						{ selected.map( ( item, i ) => {
-							const screenReaderLabel = sprintf(
-								__( '%1$s (%2$s of %3$s)', 'wc-admin' ),
-								item.label,
-								i,
-								selected.length
-							);
-							return (
-								<Tag
-									key={ item.id }
-									id={ item.id }
-									label={ item.label }
-									remove={ this.removeResult }
-									removeLabel={ __( 'Remove product', 'wc-admin' ) }
-									screenReaderLabel={ screenReaderLabel }
+					{ ( { listBoxId, activeId, onChange } ) =>
+						// Disable reason for no-static-element-interactions: There is no appropriate role
+						// which describes the input container intended accessible usability.
```

Comment:
Can you rephrase this a little? Actually you could probably merge both disable reasons into one, since it's really the same for both, right?